runtime/src/main/java/org/corfudb/runtime/view/ChainReplicationView.java

```diff
@@ -119,7 +121,8 @@ public void fillHole(long address) throws OverwriteException {
         int numUnits = getLayout().getSegmentLength(address);
         for (int i = 0; i < numUnits; i++) {
             log.trace("fillHole[{}]: chain {}/{}", address, i + 1, numUnits);
-            // In chain replication, we write synchronously to every unit in the chain.
+            // In chain replication, we write synchronously to every unit in
+            // the chain.
```

Comment:
the line breaks are intentional. in the future 80 col style restriction will be introduced, so I have it set on my editor by default